src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -49,6 +54,18 @@ public CommandResult execute(String commandText) throws CommandException, ParseE
         Command command = addressBookParser.parseCommand(commandText);
         commandResult = command.execute(model);
 
+        if (!model.getFilteredCustomerList().contains(model.getSelectedCustomer().getValue())) {
+            model.updateFilteredCustomerList(PREDICATE_SHOW_ALL_CUSTOMERS);
+            List<Customer> customers = model.getFilteredCustomerList();
+            model.selectCustomer(customers.size() > 0 ? customers.get(0) : null);
+        }
+
+        if (!model.getFilteredCommissionList().contains(model.getSelectedCommission().getValue())) {
+            model.updateFilteredCommissionList(PREDICATE_SHOW_ALL_COMMISSIONS);
+            List<Commission> commissions = model.getFilteredCommissionList();
+            model.selectCommission(commissions.size() > 0 ? commissions.get(0) : null);
+        }
```

Comment:
This looks like it was added in on accident - an abstracted version of this is already in your next PR - https://github.com/AY2223S1-CS2103T-W11-3/tp/pull/98